typespec-tests/tsp/visibility.tsp

```diff
@@ -40,23 +40,31 @@ model WriteDog {
 interface VisibilityRead {
   @get
   @route("/read")
-  get(): Dog;
+  get(): {
+    @body body: Dog;
+  };
 }
 
 @operationGroup
 interface VisibilityWrite {
   @put
   @route("/write")
-  create(@body dog: WriteDog): Dog;
+  create(@body dog: WriteDog): {
+    @body body: Dog;
+  };
 }
 
 @route("/visibility")
 interface VisibilityOp extends VisibilityRead, VisibilityWrite {
   @post
   @route("/query")
-  query(@body dog: ReadDog): Dog;
+  query(@body dog: ReadDog): {
+    @body body: Dog;
+  };
 
   @put
   @route("/roundtrip")
-  roundtrip(@body body: RoundTripModel): RoundTripModel;
+  roundtrip(@body body: RoundTripModel): {
+    @body body: RoundTripModel;
```

Comment:
If no `@body` added, it will be interpreted as no response body?

Reply:
Wonder why some are not affected..
https://github.com/Azure/autorest.java/blob/6df932a061ef99f88eaede77840efa94d221e948/typespec-tests/tsp/enum.tsp#L93-L95

Reply:
model is
```
model RoundTripModel {
  name: string;
  @visibility("create", "update") secretName: string;
}
```

If no explicit body, the `secretName` will not be in response body (as the visibility), and this "response" would be anonymous model.

Reply:
> Wonder why some are not affected..
> 
> https://github.com/Azure/autorest.java/blob/6df932a061ef99f88eaede77840efa94d221e948/typespec-tests/tsp/enum.tsp#L93-L95

I guess `Operation` does not have `@visibility`.